plugins/modules/azure_rm_recoveryservicesvault.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020 Suyeb Ansari (@suyeb786)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1', 'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = \
+    '''
+---
+module: azure_rm_recoveryservicesvault
+version_added: '1.1.0'
+short_description: Create and Update Azure Recovery Services vault
+description:
+    - Create Azure Recovery Services vault.
+    - Update Azure Recovery Services vault.
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: true
+        type: str
+    recovery_service_vault_name:
```

Comment:
I think in the module, you can use the name directly, which is concise and clear, Synchronize changes below.
```suggestion
    name:
```

Reply:
Hello @Fred-sun ,

The code is adjusted accordingly.